utility/src/main/java/org/oppia/util/statusbar/StatusBarColor.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.util.statusbar
+
+import android.os.Build
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+
+class StatusBarColor {
```

Comment:
Add a Kdoc for this class explaining what it does.

Reply:
+1

Reply:
Done

Reply:
+1